x-pack/platform/plugins/shared/observability_ai_assistant/common/conversation_complete.ts

```diff
@@ -112,8 +113,8 @@ export enum ChatCompletionErrorCode {
   InternalError = 'internalError',
   NotFoundError = 'notFoundError',
   TokenLimitReachedError = 'tokenLimitReachedError',
-  FunctionNotFoundError = 'functionNotFoundError',
   FunctionLimitExceededError = 'functionLimitExceededError',
+  FunctionArgsValidationError = 'functionArgsValidationError',
```

Comment:
Isn't the inference plugin handling this? Do we need our own error codes for this?